[raylib] msys2/mingw64 build failure

**Host Environment**
- OS: Windows (MinGW64)
- Compiler: gcc (Rev2, Built by MSYS2 project) 10.3.0

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install raylib:x64-mingw-dynamic`

**Failure logs** 
>C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lasan
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lubsan
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.

**Additional context**
It appears that the portfile for raylib is trying to use 'asan' and 'ubsan' which aren't available in MinGW, editing the port file to set '-DENABLE_ASAN' (line 44) to OFF and the same with '-DENABLE_UBSAN' (line 45) allows raylib to build.

--- a/ports/raylib/portfile.cmake
+++ b/ports/raylib/portfile.cmake
@@ -30,6 +30,12 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         use-audio USE_AUDIO
 )
 
+if(VCPKG_TARGET_IS_MINGW)
+    set(DEBUG_ENABLE_SANITIZERS OFF)
+else()
+    set(DEBUG_ENABLE_SANITIZERS ON)
+endif()
+
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
@@ -41,8 +47,8 @@ vcpkg_configure_cmake(
         -DUSE_EXTERNAL_GLFW=OFF # externl glfw3 causes build errors on Windows
         ${FEATURE_OPTIONS}
     OPTIONS_DEBUG
-        -DENABLE_ASAN=ON
-        -DENABLE_UBSAN=ON
+        -DENABLE_ASAN=${DEBUG_ENABLE_SANITIZERS}
+        -DENABLE_UBSAN=${DEBUG_ENABLE_SANITIZERS}
         -DENABLE_MSAN=OFF
     OPTIONS_RELEASE
         -DENABLE_ASAN=OFF

--- a/ports/raylib/vcpkg.json
+++ b/ports/raylib/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "raylib",
   "version-semver": "3.7.0",
+  "port-version": 1,
   "description": "A simple and easy-to-use library to enjoy videogames programming",
   "homepage": "https://github.com/raysan5/raylib",
   "supports": "!(arm | uwp)",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5426,7 +5426,7 @@
     },
     "raylib": {
       "baseline": "3.7.0",
-      "port-version": 0
+      "port-version": 1
     },
     "rbdl": {
       "baseline": "2.6.0",

--- a/versions/r-/raylib.json
+++ b/versions/r-/raylib.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "e27352fbab2a4e815a478265032a4faa60d34060",
+      "version-semver": "3.7.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "197576ef49480a84f0d5a4d153a1d9416b8230fc",
       "version-semver": "3.7.0",
       "port-version": 0

